logic/metrics/metrics.go

```diff
@@ -102,7 +102,7 @@ func GetExchangedBytesForNode(node *models.Node, metrics *models.Metrics) error
 		return err
 	}
 	defer wgclient.Close()
-	device, err := wgclient.Device(node.Interface)
+	device, err := wgclient.Device("netmaker")
```

Comment:
netmaker should be a constant

Reply:
added new constant; applied to other instances as well